data/siggraph-2019/10.1145-3306346.3323032.json

```diff
@@ -0,0 +1,207 @@
+        "##": "topic classification.",
+        "Topic {Rendering, Animation and Simulation, Geometry, Images, Virtual Reality, Fabrication}": "Rendering",
+
+        "##": "Is any of the co-authors from a university of a public institution ? (true/false)",
+        "Co-authors from academia (boolean)": true,
+
+        "##": "Is any of the co-authors from a private company? (true/false)",
+        "Co-authors from industry (boolean)": false,
+
+        "##": "Is the paper available as ACM Open Access? (true/false) ",
+        "ACM Open Access (boolean)": false,
+
+        "##": "PDF accessibility on the author or project or institution web page (true/false)",
+        "PDF on the authors' webpage / institution (boolean)": false,
+
+        "##": "Is a paper preprint available on Arxiv or any Open Archive Initiative",
+        "##": "platform (true/false) ?",
+        "PDF on Arxiv or any openarchive initiatives (boolean)": false,
+        "Arxiv/OAI page URL": "",
+
+        "##": "URL for direct download of the PDF paper (eiher author/institution/arxiv website)",
+        "##": "(empty if PDF not available)",
+        "PDF URL": "https://dl.acm.org/doi/pdf/10.1145/3306346.3323032",
+        
+        "##": "Project URL (if any)",
+        "Project URL": "",
+
+        "##": "Is companion code available ? (true/false)",
+        "Code available (boolean)": false,
+        "##": "Direct link to the code",
+        "Code URL": "",
+        "##": "A second direct link to the code (may be empty)",
+        "Code URL2": "",
+        "##": "If the code is given as an archive file, specify",
+        "##": "the md5 hash of the original archive",
+        "MD5 sum (for archives)": "",
+        "MD5 sum (for archives) URL2": "",
+        "##": "If the code is hosted on a git/hg or svn repository,",
+        "##": "specify the commit or revision number of the",
+        "##": "version of the code you have tested.",
+        "git/hg/svn commit hash or revision number": "",
+        "git/hg/svn commit hash or revision number URL2": "",
+
+        "##": "Is the code available on the softwareheritage.org website ?",
+        "##": "If so, please specify the permalink to the specific revision being reviewed",
+        "##": "If not, just leave this field empty.",
+        "Software Heritage permalink": "",
+       
+        "##": "If no code is provided, has the paper some included pseudo-code ? (true/false)",
+        "If code not available, pseudo-code available (boolean)": true,
+        
+        "##": "If some pseudo-code is given, would it be easy to implement the paper contribution from it ?",
+        "##": "Please add a rate between 0 and 4",
+        "##": "(0 = This implementation is not straightforward at all, many elements are missing)",
+        "##": "(4 = I can copy-paste the pseudo-code to reproduce something)",
+        "If pseudo-code, could the paper be trivially implemented? {0..4}": "4",
```

Comment:
```suggestion
        "If pseudo-code, could the paper be trivially implemented? {0..4}": 4,
```

Reply:
a bit strange that the checker did not complain about that...

Reply:
arg.. maybe the test is not applied to the pseudoscore